### PR TITLE
Add support for balenaOS prefixed OS versions

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Change-type: minor
Signed-off-by: Lucian Buzzo <lucian.buzzo@gmail.com>

### CODE REVIEW

File: test/versions.ts [hunk 1]
```diff
@@ -77,5 +77,7 @@ export const versions = [
 	'2.7.2+asdf',
 	'Resin OS 2.7.9+rev1',
 	'Balena OS 2.13.0',
+	'BalenaOS 2.13.1',
```
Comment (ZubairLK) [inline test/versions.ts]: Not sure I understand this one?

File: src/index.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ const trimOsText = (version: string) => {
 	// Remove "Resin OS" and "Balena OS" text
 	return (
 		version
-			.replace(/(resin|balena)\sos\s+/gi, '')
+			.replace(/(resin|balena)\s*os\s*/gi, '')
```
Comment (thgreasi) [inline src/index.ts]: `?` might have been a better match than `*` but being extra defensive about white spaces makes me feel more confident that this isn't going to break by accident :+1:

**Comment by LucianBuzzo** [comment]
@resin-ci retest

**Comment by LucianBuzzo** [comment]
@resin-ci retest

**Pull request merged**